core/capability-negotiation.md

```diff
@@ -208,10 +208,30 @@ When CAP Version `302` is enabled, the client also implicitly indicates support
 `cap-notify` capability listed below, and support for the relevant `NEW` and `DEL`
 subcommands.
 
+The CAP version number argument MUST be treated as a digit by the server. If the client's
```

Comment:
Hmm, it already consists of three digits, so "the CAP version argument MUST be treated as a number" would be more correct (…or "as a decimal integer", depends on how specific you want it).

Reply:
Makes sense. I'll just leave it as 'number', that should be fine